js/server/process-lm.js

```diff
@@ -53,6 +55,14 @@ function processUserEvents (users, events) {
       forfeited: 0
     };
     if (event.amount > 0) {
+      if (
+        // is after deposits are allowed
+        event.timestamp >
+        DEPOSITS_ALLOWED_DURATION_MS / 1000 / 60
```

Comment:
This needs to be floored here, would actually allow a timestamp over the last block